### PR TITLE
Updated README.md for eZ Platform

### PR DESCRIPTION
**Pull request by bdunogier**
Maybe this PR should actually be sent to ezsystems/ezplatform directly ?

## TODO
- [x] Finalize requirements change
- [x] Fix link to legacy install doc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,55 +11,54 @@ define model structures using fields, trees and relations, and has a very
...
+eZ Platform exists in two versions; this, the community version is available under
 the GPLv2 license, while several extended versions for enterprise & business is available
 under a more permissive business license, see [ez.no](http://ez.no/) for more info.
 
 ## Install, Upgrade and Getting started
-For installation & upgrade instructions, see [INSTALL.md](https://github.com/ezsystems/ezpublish-community/blob/master/INSTALL.md).
+For installation & upgrade instructions, see [INSTALL.md](https://github.com/ezsystems/ezplatform/blob/master/INSTALL.md).
 
-To get started with coding, see [GETTING_STARTED.md](https://github.com/ezsystems/ezpublish-community/blob/master/GETTING_STARTED.md).
+To get started with coding, see [GETTING_STARTED.md](https://github.com/ezsystems/ezplatform/blob/master/GETTING_STARTED.md).
```
Comment (crevillo) [inline README.md]: so, there will be a change in the repo name and ezpublish-community will gonna dissapear?

Comment (andrerom) [inline README.md]: No, we keep both, there have been some community members that want to help out making further community distributions so these changes are actually for that repo, here we will roll back a few changes to make both projects possible at the same time.

**Comment by andrerom** [comment]
> Maybe this PR should actually be sent to ezsystems/ezplatform directly ?

yes

anyway +1 :)

**Comment by bdunogier** [comment]
Closing, and reopening on ezsystems/ezplatform.

**Pull request closed without merging**